manifests/autoheal-deployment.yml

```diff
@@ -29,6 +30,20 @@ spec:
         app: autoheal
     spec:
       serviceAccountName: autoheal
+      volumes:
+      - name: config
+        configMap:
+          name: autoheal
       containers:
       - name: autoheal
         image: openshift/origin-autoheal:latest
+        imagePullPolicy: IfNotPresent
+        volumeMounts:
+        - name: config
+          mountPath: /etc/autoheal
+        command:
+        - /usr/bin/autoheal
+        args:
+        - server
```

Comment:
what is the `server` arg?

Reply:
We need to provide this arg now:
```
Usage:
  autoheal [command]

Available Commands:
  help        Help about any command
  server      Starts the auto-heal server
```

Reply:
In that case, I think we should make the server command default, and display help when `--help` flag is provided. Opened https://github.com/openshift/autoheal/issues/22 to track this